.env.defaults

```diff
@@ -33,12 +32,14 @@ CURVEBALL_ORIGIN="http://localhost:8531"
 # DB_DATABASE=a12nserver
 
 # Postgres database settings
-#
-# DB_DRIVER=postgres
-# DB_HOST=127.0.0.1
-# DB_PASSWORD='password
-# DB_USER=a12nserver
-# DB_DATABASE=a12nserver
+DB_PORT=5342
+DB_DRIVER=postgres
+DB_HOST=127.0.0.1
+DB_PASSWORD=password
+POSTGRES_PASSWORD=password
+DB_USER=a12nserver
+DB_DATABASE=a12nserver
+DATABASE_URL=postgres://a12nserver:password@127.0.0.1:5342/a12nserver
```

Comment:
The old values were correct, there's no `DATABASE_URL` environment variable, and `POSTGRESS_PASSWORD` is an environment variable that exists on the standard docker postgres container, but it's not an a12n-server environment var so it doesn't really belong here.